packages/runtime-common/code-ref.ts

```diff
@@ -227,3 +227,27 @@ export function humanReadable(ref: CodeRef): string {
 function assertNever(value: never) {
   return new Error(`should never happen ${value}`);
 }
+
+async function isInsideAncestorChainOfCardDef(
```

Comment:
javascript has a similar kind of function called `typeof`. maybe that is a clearer name for this method: `isTypeOf()`

Reply:
Ahh ok I think this worked but loadCard is async so I used .then within the isCardDef implementation. I also added a note on using `isCardDef`